amundsen_application/api/utils/metadata_utils.py

```diff
@@ -150,16 +149,43 @@ def marshall_dashboard_full(dashboard_dict: Dict) -> Dict:
     return dashboard_dict
 
 
-def _update_prog_descriptions(prog_descriptions: List) -> None:
+def _convert_prog_descriptions(prog_descriptions: List = []) -> Dict:
```

Comment:
For good practice, we don' use [] for default argument (https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments) but instead just use None for default.

Reply:
I see, updating this before merging because I see how it can cause a bug.